scintellometry/folding/fold.py

```diff
@@ -174,6 +174,8 @@ def fold(fh, comm, samplerate, fedge, fedge_at_top, nchan,
         # same as fine = rfftfreq(2*ntint, dtsample.value/2.)[::2]
         fcoh = freq_in[np.newaxis, :] + tb * u.Hz * rfftfreq(
             ntint*2, dtsample.value/2.)[::2, np.newaxis]
+        fcoh = freq_in - u.Hz * np.fft.fftfreq(ntint, dtsample.value)[:,np.newaxis]
```

Comment:
@ramain - why is the definition of `fcoh` overridden here?  Maybe just remove the first one?  Also, do include the `tb`. Since for ARO, one has `fedge_at_top is True`, it should just be `+ tb * u.Hz * ...`. 
Also, PEP8 would want a space after the `,` before `np.newaxis`...